SECURITY.md

```diff
@@ -1,11 +1,142 @@
-# Security
+# Aptos Core Bug Bounty
 
-As Aptos Core is currently in the prototype stage and does not power a blockchain with a cryptocurrency that has real-world value, our security procedures are not fully in place. We will release a more complete disclosure policy prior to the launch of the mainnet.
+## Reporting a Security Concern
 
-## Reporting a Bug
+**DO NOT CREATE AN ISSUE** to report a security problem.
 
-The Aptos Core team and community take all security bugs in the Aptos project seriously to ensure the security of the Aptos Blockchain.
+Send an email to [security@aptosfoundation.org](mailto:security@aptosfoundation.org) and provide your GitHub username. The team will create a new draft security advisory for further discussion.
```

Comment:
nit: Maybe make it more obvious that this will occur in a private github repo and they'll be given access, e.g.:

```
Send an email to [security@aptosfoundation.org](mailto:security@aptosfoundation.org) and provide your GitHub username. The team will create a new draft security advisory in a private github repository and provide you access for further discussion.
```